Library/Homebrew/style.rb

```diff
@@ -172,8 +172,6 @@ def run_shellcheck(files, output_type)
         "--enable=all",
         "--external-sources",
```

Comment:
```suggestion
```

If this is no longer needed?

Reply:
I think it is still needed for `brew style <file>`, such as using `brew style install.sh` for Homebrew/install.

Reply:
Ok, cool 👍🏻